traffic_ops/traffic_ops_golang/server/servers.go

```diff
@@ -855,22 +893,12 @@ func getServers(h http.Header, params map[string]string, tx *sqlx.Tx, user *auth
 
 	// TODO there's probably a cleaner way to do this by preparing a NamedStmt first and using its QueryRow method
 	var serverCount uint64
-	countRows, err := tx.NamedQuery(serverCountQuery+queryAddition+where, queryValues)
+	serverCount, err = getServerCount(tx, serverCountQuery+queryAddition+where, queryValues)
 	if err != nil {
 		return nil, 0, nil, fmt.Errorf("failed to get servers count: %v", err), http.StatusInternalServerError, nil
 	}
-	defer countRows.Close()
-	rowsAffected := 0
-	for countRows.Next() {
-		if err = countRows.Scan(&serverCount); err != nil {
-			return nil, 0, nil, fmt.Errorf("failed to read servers count: %v", err), http.StatusInternalServerError, nil
-		}
-		rowsAffected++
-	}
-	if rowsAffected != 1 {
-		return nil, 0, nil, fmt.Errorf("incorrect rows returned for server count, want: 1 got: %v", rowsAffected), http.StatusInternalServerError, nil
-	}
 
+	serverCount += serverCountOrigin
```

Comment:
Tested this against CDN in a Box. In the case of no required capabilities, the original server count (6) is accurate, so the reported server count ends up being 7 instead of 6.

Reply:
Fixed the count now. I'm querying for the ORGs separately only when its specific to a DS that has reqd capabilities.